githubchristophersummitt-warren.md

```diff
@@ -0,0 +1,7 @@
+## Warren, Ohio
+
+### Great places to eat
```

Comment:
@christophersummitt where is Warren Ohio in relation to Akron? Do you have any good places to recommend that serve Italian?